tools/lkl/lib/virtio_net_dpdk.c

```diff
@@ -33,9 +33,14 @@ static char *ealargs[4] = {
 /* XXX: disable cache due to no thread-safe on mempool cache. */
 #define MEMPOOL_CACHE_SZ        0
 /* for TSO pkt */
-#define MAX_PACKET_SZ           (65535 \
+//For ixgbe and vmxnet3 drivers
+/*#define MAX_PACKET_SZ           (65535 \*/
+/*	- (sizeof(struct rte_mbuf) + RTE_PKTMBUF_HEADROOM))*/
+//For Intel i350 NIC (igb driver)
+#define MAX_PACKET_SZ           (16383 \
 	- (sizeof(struct rte_mbuf) + RTE_PKTMBUF_HEADROOM))
-#define MBUF_NUM                (512*2) /* vmxnet3 requires 1024 */
+//#define MBUF_NUM                (512*2) /* vmxnet3 requires 1024 */
+#define MBUF_NUM                4096 /* Intel i350 NIC (igb driver): 4096 */
```

Comment:
I am not very happy with these hardcoded numbers.

Are these performance optimizations or limitations of the nic?

Reply:
I agree with @tavip : let LKL_HIJACK_NET_IFPARAMS for DPDK iftype to configure those parameters.

Reply:
Thank you for feedback. I will implement the indicated changes asap.